hw/cdc/tools/verixcdc/run-cdc.tcl

```diff
@@ -114,33 +114,6 @@ set_user_cntl_synchronizer -name opentitan_2ff $prim_2ff_modules
 # TODO: These should not be hardcoded here, instead, we need to create another variable called CDC_CONSTRAINT
```

Comment:
Is this comment still needed, now that all these custom paths are gone?

Maybe it makes sense to move it further down below before this statement: 
```systemverilog
puts $outfile "set_stable {{top_earlgrey.u_pinmux_aon.dio_out_retreg_q[15:0]}}"
```

Reply:
It's fixed. Thanks